graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -257,7 +296,14 @@ def _rename_name_and_add_to_record(self, node):
             node_with_new_name = get_copy_of_node_with_new_name(node, new_name_string)
             return node_with_new_name
 
-    def enter(self, node, key, parent, path, ancestors):
+    def enter(
+        self,
+        node: Node,
+        key: Optional[Any],
```

Comment:
type annotations like these might seem strange since Any includes None, but [it's a bit more complicated](https://github.com/python/mypy/issues/3138) as a consequence of how unions work